tests/testthat/test-04-model-frame.R

```diff
@@ -53,7 +53,26 @@ test_that("artnum_mf() works with single quarter ART data", {
 })
 
 
+test_that("Informative error displayed when model run to admin level higher/lower than population data supplied", {
+  x <- expect_error(
+    naomi_model_frame(a_area_merged,
+                      demo_population_agesex,
+                      a_spec,
+                      scope = "MWI",
+                      level = 3,
+                      calendar_quarter1 = "CY2016Q1",
+                      calendar_quarter2 = "CY2018Q4",
+                      calendar_quarter3 = "CY2019Q2",
+                      calendar_quarter4 = "CY2022Q3",
+                      calendar_quarter5 = "CY2023Q3"))
 
+  expect_equal(
+    x$message,
+    paste("Population data not available for admin level selected",
+          "for model projections. Please review model options",
+          "selection to ensure that area level selection is correct.")
+  )
+})
```

Comment:
Could the error message here quote the area level and area level label selected?

> "Population data not available for selected area level 3 (District). Pleas review model options"

Also I'm think I would remove the "for model projections". We usually use "projections" to refer to the T3/T4/T5 projection, but here this area level selection pertains to everything.